src/BlackSlope.Api/Startup.cs

```diff
@@ -85,17 +85,8 @@ private void CorsConfiguration(IServiceCollection services)
             });
         }
 
-        private void AuthenticationConfiguration(IServiceCollection services)

```

Comment:
This was the cleanup piece right?

Reply:
yes, it has been moved into common as AddAzureAd extension method for ServiceCollection.